contribs/gmf/less/map.less

```diff
@@ -67,39 +67,41 @@ button[ngeo-mobile-geolocation] {
   list-style: none;
 
   li {
-    width: 100%;
     cursor: pointer;
-    padding: @app-margin 0;
-    color: @color;
-  }
-
-  .gmf-check,
-  .gmf-text {
-    float: left;
-  }
+    display: flex;
+    align-items: center;
+    min-height: 30px;
```

Comment:
Because the thumbnails are images. The size of the images are given in pixels. This value prevents the blank background layer to be displayed with a completely different layout.